<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>final_activation_dialog</class>
 <widget class="QDialog" name="final_activation_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>最终激活</string>
  </property>
  <widget class="QLabel" name="difficulty_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>激活难度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="die_result_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="roll_button">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>掷骰</string>
   </property>
  </widget>
  <widget class="QLabel" name="die_result_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="heart_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heart_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heart_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heart_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heart_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heart_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources/resources.qrc">:/full_heart</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
